app/controllers/posts_controller.rb

```diff
@@ -1,15 +1,35 @@
 class PostsController < ApplicationController
-  before_action :set_user
-
   def index
-    @posts = Post.all
+    @user = User.find(params[:user_id])
+    @posts = @user.posts.order(created_at: :asc)
+  end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @post = Post.new(post_params)
+    @post.author = current_user
+    @post.comments_counter = 0
+    @post.likes_counter = 0
+    if @post.save
+      flash[:notice] = 'Post created successfully!'
+      redirect_to user_posts_path(current_user)
+    else
+      flash[:alert] = "Couln't create post!"
+      render :new, status: :unprocessable_entity
+    end
```

Comment:
- [ ] Great job for adding the new and create but I can't see any forms in your views to let me be able to create a new post and I can't see the users even if I am putting the right route in the URL but I am not be able to see any posts it shows me this error could you solve it please? 💯🔥🤩✨

## I created some users and posts with `rails c` but I can't see ⬇⬇
![image](https://user-images.githubusercontent.com/58553711/222934494-f155f303-cc80-4dd9-a380-55503867797f.png)

Reply:
there is not any user in your database with id = 1. I do not get any errors all features are working.
![users](https://user-images.githubusercontent.com/60324033/222935543-bd2bd6a9-b027-45ea-833a-b64a46824439.PNG)

Reply:
please compare my design with the reference design, I just followed the reference design. 
![users](https://user-images.githubusercontent.com/60324033/222935790-949734d5-6b22-4052-9e84-5ca08ace72cd.PNG)
![single-user](https://user-images.githubusercontent.com/60324033/222935796-50fb6b99-bd06-4ed7-b864-eda5c7436f90.PNG)
![post](https://user-images.githubusercontent.com/60324033/222935819-6d44c332-75e9-4ee3-905c-f74f6b1f82df.PNG)
![single-post](https://user-images.githubusercontent.com/60324033/222935860-efa596b9-1629-4002-ae6e-f271e5c46526.PNG)
![reference](https://user-images.githubusercontent.com/60324033/222935928-b30eaa2a-7518-4d66-ba7a-cbdc56e10865.PNG)